src/main/java/edu/sybit/codingcamp/battleship/websocket/WebSocketMappings.java

```diff
@@ -60,13 +60,16 @@ public void gamefield(Message message) {
             LOGGER.debug("no Match found -> create new one.");
             match = matchService.createNewMatch(message.getMatchId(), player);
         }
-        
+
         if(!(player.equals(match.getPlayer1())) && (match.getPlayer2() == null)) {
             //set second player
-            matchService.addOpponentPlayer(match, player);
-            
+            matchService.addOpponentPlayer(match, player);         
```

Comment:
Ich benutze hier wieder die addOpponent anstatt die addNewPlayer, die in dem PR neu war. Dann braucht die Methode auch gar keine Exception schmeißen.